### PR TITLE
MTM-36743 added info for smartrest via mqtt

### PR DESCRIPTION
**Pull request by rssag**

### CODE REVIEW

File: content/device-sdk/mqtt-bundle/smartrest-1.md [hunk 1]
```diff
@@ -20,7 +20,16 @@ Although you need to send the IDs in the body of each message with SmartREST 1.0
 
 The MQTT ClientId needs to match the externalId with type **c8y_Serial** of your device. It is used to assign the correct operations and responses.
 
-### Sending SmartREST 1.0
+### Sending and receiving SmartREST 1.0
+
+In general, the following hold for SmartREST requests and responses via MQTT:
```
Comment (BeateRixen) [inline content/device-sdk/mqtt-bundle/smartrest-1.md:25]: hold > holds

File: content/device-sdk/mqtt-bundle/smartrest-1.md [hunk 2]
```diff
@@ -20,7 +20,16 @@ Although you need to send the IDs in the body of each message with SmartREST 1.0
...
+
+* All request rows will be sent as single MQTT messages. A single request message always yields a single response message instead of being split up to several response messages
```
Comment (MWindel) [inline content/device-sdk/mqtt-bundle/smartrest-1.md:27]: Add either comma or full stop to keep layout consistent with other bulleted lists in Device SDK guide

File: content/device-sdk/mqtt-bundle/smartrest-1.md (hunk 1 above)
Comment (rssag) [reply content/device-sdk/mqtt-bundle/smartrest-1.md]: done

File: content/device-sdk/mqtt-bundle/smartrest-1.md (hunk 2 above)
Comment (rssag) [reply content/device-sdk/mqtt-bundle/smartrest-1.md]: done

**Review by MWindel — APPROVED** [review]

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**